airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -1010,6 +1010,81 @@ definitions:
       $parameters:
         type: object
         additionalProperties: true
+  SessionTokenAuthenticator:
+    type: object
+    required:
+      - type
+      - login_requester
+      - session_token_path
+      - request_authentication
+    properties:
+      type:
+        type: string
+        enum: [SessionTokenAuthenticator]
+      login_requester:
+        title: Login Requester
+        description: Description of the request to perform to obtain a session token to perform data requests. The response body is expected to be a JSON object with a session token property.
+        "$ref": "#/definitions/HttpRequester"
+        examples:
+          - type: HttpRequester
```

Comment:
I wonder how long it'll take before we start seeing nested SessionTokenAuthenticators...